lib/stupidedi/versions/004010/element_defs.rb

```diff
@@ -9461,6 +9466,71 @@ module ElementDefs
           s::CodeList.external("559"))
         E1715 = t::ID.new(:E1715, "Country Subdivision Code"             , 1, 3,
           s::CodeList.external("5"))
+        E1716  = t::AN.new(:E1716 , "Change Order Sequence Number"       , 1, 8)
+        E1717  = t::ID.new(:E1717 , "Allowance or Charge Indicator"      , 1, 1,
+                           s::CodeList.build(
+                               "A"   => "Allowance",
```

Comment:
The `=>` tokens line up vertically, but you don't need the extra preceding spaces.